### ISSUE TITLE
Not work in Firefox 57

### ISSUE THREAD

**Issue by sbruselario** [bakulf/switchy#6]
The extension "not meet current Firefox standards" and has been deactivated by the browser when upgrade to Firefox 57.

**Comment by Delerium69** [comment]
Yep :( doesn't work with Firefox 57 - i adored this addon as i use multiple instances/profiles. Any chance of it getting updated?
